treeherder/model/migrations/0052_remove_unnecessary_job_indexes.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-01-25 18:01
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0051_add_job_lastmodified_index'),
+    ]
+
+    operations = [
```

Comment:
And yeah, I noticed django seemed to be smart enough not to add the guid index in the first place, though I figured it couldn't hurt to kill the index property.